helmcharts/etcd/templates/secret-etcd-backup.yaml

```
@@ -19,6 +19,8 @@ metadata:
   name: {{ .Values.name }}-backup
   namespace: {{ .Release.Namespace }}
 type: Opaque
+stringData:
+{{ toYaml .Values.backup.secretStringData | indent 2 }}
```

```suggestion
{{- toYaml .Values.backup.secretStringData | nindent 2 }}
```
(strips whitespaces first, then new-line idents by 2)

let's be more error-resistant